### PR TITLE
Task dev 5

### PR DESCRIPTION
**Pull request by VladimirVolkovD**
Providing DEV5 for review.

### CODE REVIEW

File: DEV-5/ProductStorage.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace DEV5
+{
+    /// <summary>
+    /// Determines the function for Storage classes.
+    /// </summary>
```
Comment (IgorKheidorov) [inline DEV-5/ProductStorage.cs:7]: Why generic if you do not use parameter T

File: DEV-5/Product.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿namespace DEV5
+{
+    /// <summary>
+    /// Represents entity of product for ProductStorage class.
+    /// </summary>
```
Comment (IgorKheidorov) [inline DEV-5/Product.cs:5]: Abstract class is necessary if at least has one field or method

File: DEV-5/EntryPoint.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿namespace DEV5
+{
+    /// <summary>
+    /// It's an entry point in the program.
+    /// </summary>
+    class EntryPoint
+    {    
+        static void Main(string[] args)
+        {
+            CarsStorage carsStorage = new CarsStorage();
```
Comment (IgorKheidorov) [inline DEV-5/EntryPoint.cs:10]: I do not see here any functionality, it looks like that all functionality is inside constructor

File: DEV-5/AllCounter.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace DEV5
+{
+    /// <summary>
+    /// AllCounter implements method for command selection of specific type for object being transferred.
+    /// </summary>
```
Comment (IgorKheidorov) [inline DEV-5/AllCounter.cs:7]: ICountable

File: DEV-5/Car.cs [hunk 5]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading;
+
+namespace DEV5
+{
+    /// <summary>
+    /// Represents entity of car.
+    /// </summary>
+    public class Car : Product
+    {
```
Comment (IgorKheidorov) [inline DEV-5/Car.cs:10]: Upper case first letter for properties

File: DEV-5/Car.cs [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        public string brand { get; }
+        public string model { get; }
```
Comment (IgorKheidorov) [inline DEV-5/Car.cs:12]: quantity has not to be a field of object car, this is very bad idea

**Review by IgorKheidorov — APPROVED** [review]
Pattern command is not applied

**Comment by VladimirVolkovVD** [comment]
This task was complet in task_DEV-8

**Pull request closed without merging**